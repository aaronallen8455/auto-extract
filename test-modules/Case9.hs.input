module Case9 where

z :: Integer
z = 9

test =
  let y = 5
      x = 3
      k = 3
      l = 2
      i = 9
      h = 20
      g = 21
      j = 8
   in EXTRACT@a (x + y * z + k + l + i + h + g + j)
