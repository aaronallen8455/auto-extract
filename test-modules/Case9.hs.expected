module Case9 where

z :: Integer
z = 9

test =
  let y = 5
      x = 3
      k = 3
      l = 2
      i = 9
      h = 20
      g = 21
      j = 8
   in a g h i j k l x y

a :: Integer -> Integer -> Integer -> Integer -> Integer -> Integer -> Integer -> Integer -> Integer
a g h i j k l x y = x + y * z + k + l + i + h + g + j
